services/brig/src/Brig/Index/Options.hs

```diff
@@ -95,12 +97,13 @@ makeLenses ''CassandraSettings
 
 makeLenses ''ReindexFromAnotherIndexSettings
 
-mkCreateIndexSettings :: ElasticSettings -> ([ES.UpdatableIndexSetting], Int)
+mkCreateIndexSettings :: ElasticSettings -> ([ES.UpdatableIndexSetting], Int, Maybe ES.TemplateName)
```

Comment:
This tuple was already weird now it has 3 things, I think it is worth creating a type with appropriate names for these things. But I don't feel too strongly of this, hoping that it will be deleted at some point.

Reply:
Created a data type for this